<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecodeSlatepackDlg</class>
 <widget class="QDialog" name="DecodeSlatepackDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="control::MwcPushButtonNormal" name="closeButton">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>515</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>14000</width>
     <height>40</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="slatepackEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>85</y>
     <width>401</width>
     <height>406</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>6000</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Please paste the slatepack content below</string>
   </property>
  </widget>
  <widget class="control::MwcLabelLarge" name="title">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>25</y>
     <width>701</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Decode Slatepack</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="slateJsonEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>85</y>
     <width>401</width>
     <height>406</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>6000</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Slate Json</string>
   </property>
  </widget>
  <widget class="control::MwcLabelNormal" name="slateDetailsLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Slate Details</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
